ISISScientificComputing/autoreduce
# 418 query computation

### Summary of work
Subsegment of system performance package to handle query argument specification and data manipulations to data queried from database.

### How to test your work
* Ensure code is readable
* Ensure all unit tests pass
* Ensure there is no drop in coverage

Fixes #418 

**Before merging ensure the release notes have been updated**
